components/automate-cli/cmd/chef-automate/initConfigHaTmpl.go

```diff
@@ -52,6 +52,9 @@ admin_password = ""
 # Automate Load Balancer FQDN eg.: "chefautomate.example.com"
 fqdn = ""
 
+# Add Automate Load Balancer root-ca
```

Comment:
I think this also needs to be added to Docs also in Sample Config and other places

Reply:
will raise separate PR for this